website/docs/guides/markdown.md

```diff
@@ -199,11 +315,54 @@ world
 
 </ChainedSnippets>
 
+## `using` directives and markdown code blocks
+
+It is possible to define `using` directives at the beginning of a `scala` code block inside a markdown input.
```

Comment:
Add an explanation, what happens if users pass multiple times the same directive in different code blocks. For example:

## `scala raw` example
 ```scala raw
//> using scala "3"
 //> using lib "com.lihaoyi::os-lib:0.8.1"
 println(os.pwd)
 ```

 ## Plain `scala` example
 ```scala
//> using scala "2.13"
 //> using lib "com.lihaoyi::os-lib:0.8.1"
 println(os.pwd)
 ```

Reply:
should be fixed